tests/test_oic.py

```diff
@@ -182,19 +181,14 @@ def test_access_token_request(self):
         assert _eq(resp.keys(), ["token_type", "state", "access_token", "scope"])
 
     def test_access_token_request_with_custom_response_class(self):
```

Comment:
Please leave this test unchanged. This is here to ensure that the custom response class is honored and is not actually related to the content of the response.

Reply:
Done